modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItAggregatesTest.java

```diff
@@ -617,14 +613,15 @@ public void testAvgRandom() {
             numbers.add(num);
 
             String query = "INSERT INTO numbers (id, int_col, dec10_2_col) VALUES(?, ?, ?)";
-            sql(query, i, num.intValue(), num);
+            sql(query, i, NumberUtil.rescaleBigDecimal(num, 0).intValue(), num);
         }
 
         BigDecimal avg = numbers.stream()
                 .reduce(new BigDecimal("0.00"), BigDecimal::add)
-                .divide(BigDecimal.valueOf(numbers.size()), MathContext.DECIMAL64);
+                .divide(BigDecimal.valueOf(numbers.size()), 2, RoundingMode.HALF_UP);
 
         for (String[] rules : makePermutations(DISABLED_RULES)) {
+            System.out.println(String.join(",", rules));
```

Comment:
I think this `system.out.println` is unnecessary.

Reply:
When I got a the following numbers by the random generator:  `{37, 46, 72, 33, 33, 57, 76, 27, 100, 4, 62, 77, 74, 49, 31, 16, 56, 34, 82};`

A got a test failure:

```
Expected: 50 <class java.lang.Integer>
Actual:   51 <class java.lang.Integer>
```
Query results:
```
[[51, 50.84]]
```

Looks like using avg.intValue() ` .returns(avg.intValue(), avg)` is no longer a valid check?